### PR TITLE
Update docs for the order

### PR DESCRIPTION
**Pull request by KristopherJordan**
Updated some docs regarding the Order

### CODE REVIEW

File: source/includes/_orders.md [hunk 1]
```diff
@@ -16,43 +16,24 @@ deliveries | `array` | Used to store the delivery history for an order
...
-If the `price_change_percentage` is 0, the price in the sub-product adds that price to the `total_amount`.
-
-
-
-### Resources in the order
-The order could have [`resources`](#resources) associated to it.
-The `total_amount` is affected by every `retail` resource in the array, by the
-field `retail_price`.
+units | `number` | Number of unique products in an order
+stripe_charge_id | `string` | The payment charge ID in <a href="http://www.stripe.com" target="_blank">Stripe</a>
```
Comment (pedropregueiro) [inline source/includes/_orders.md]: this is not used anymore

File: source/includes/_orders.md [hunk 2]
```diff
@@ -16,43 +16,24 @@ deliveries | `array` | Used to store the delivery history for an order
...
-
-
-
-### Resources in the order
-The order could have [`resources`](#resources) associated to it.
-The `total_amount` is affected by every `retail` resource in the array, by the
-field `retail_price`.
+units | `number` | Number of unique products in an order
+stripe_charge_id | `string` | The payment charge ID in <a href="http://www.stripe.com" target="_blank">Stripe</a> 
+stripe_refund_id | `string` | The payment refund ID in <a href="http://www.stripe.com" target="_blank">Stripe</a>
```
Comment (pedropregueiro) [inline source/includes/_orders.md]: neither this

File: source/includes/_orders.md [hunk 3]
```diff
@@ -16,43 +16,24 @@ deliveries | `array` | Used to store the delivery history for an order
...
+accounting_reference | `string`| A accounting reference ID
+order_status | `string` | See [`order statuses`](#order_status)
+delivery_status | `string` | See [`delivery statuses`](#delivery_status)
```
Comment (pedropregueiro) [inline source/includes/_orders.md]: this is also above already

**Comment by KristopherJordan** [comment]
I fixed your comments @pedropregueiro. I'll merge it

**Pull request merged**